src/services/analytics.ts

```diff
@@ -14,3 +18,32 @@ export const fetchAnalyticsData = async (
     }
   )
 }
+
+export const fetchLeadsAnalytics = async ({
+  dealerId,
+  dimensions,
+  query,
+}: {
+  dealerId: number
+  dimensions?: string[]
+  query?: { verificationDateFrom: string }
+}): Promise<LeadsAnalytics> => {
+  return postData(Service.ANALYTICS, `dealers/${dealerId}/leads/analytics`, {
+    function: "count",
+    dimensions,
+    query,
+  })
+}
+
+export const fetchListingsAnalytics = async ({
+  dealerId,
+  dimensions,
+}: {
+  dealerId: number
+  dimensions?: string[]
```

Comment:
is it worth typing the dimensions here? or is it an undefined set of strings?

Reply:
I need it in case we want to pass the dimension like `conditionType`